### ISSUE TITLE
No symbols have been loaded for this document.

### ISSUE THREAD

**Issue by davismj** [Microsoft/TypeScript#7331]
**TypeScript Version:** 

1.8.0-beta

TypeScript for Visual Studio 1.8.4

**Expected behavior:**
Clicking the "Debug" button, particularly when set to "Internet Explorer", should launch Internet Explorer and enable breakpoints from Visual Studio.

**Actual behavior:** 
Though Internet Explorer opens the website and Visual Studio enters debugging mode, the "No symbols have been loaded for this document" error appears over empty circle breakpoints.

If set to Google Chrome mode, the debugging mode immediately exits with no message.

**Install Logs**
[dd_TypeScript_Dev14_20160301093256](https://www.dropbox.com/s/7euqqtmbkzvyw96/dd_TypeScript_Dev14_20160301093256.log?dl=0)
[dd_TypeScript_Dev14_20160301093256_000_TypeScript_SDK.msi](https://www.dropbox.com/s/ahb5adqwjiuoope/dd_TypeScript_Dev14_20160301093256_000_TypeScript_SDK.msi.log?dl=0)
[dd_TypeScript_Dev14_20160301093256_001_TypeScript_Dev14.msi](https://www.dropbox.com/s/qqbdmn691uwvmdn/dd_TypeScript_Dev14_20160301093256_001_TypeScript_Dev14.msi.log?dl=0)

**Comment by mhegazy** [comment]
What kind of project is it? how can i reproduce this locally?

**Comment by davismj** [comment]
Same repo: [https://github.com/foursails/visual-aurelia/](https://github.com/foursails/visual-aurelia/)

**Comment by davismj** [comment]
Is there any more information I can provide to help?

**Comment by mhegazy** [comment]
i need to debug through it again. my apologies did not have time since you posted the repro project.

**Comment by davismj** [comment]
No problem at all! I appreciate your help. Just want to make sure you've got everything you need for now.

**Comment by mhegazy** [comment]
a few notes, you do not have "sourceMap" property enabled in your tsconfig.json, so no source maps are generated, so makes sense that debugging is not working.

second, i can not seem to get the app running, i keep getting an error:

```txt
Unhandled promise rejection Error: XHR error (404 Not Found) loading http://localhost:60331/dist/app.html
	Error loading http://localhost:60331/dist/app.html!http://localhost:60331/jspm_packages/github/systemjs/plugin-text@0.0.3.js
	Error loading http://localhost:60331/dist/app.html!template-registry-entry
```

**Comment by davismj** [comment]
Sorry, I forgot to push the latest. There were a number of issues that have been resolved.

Note that I have the following in the config, which should be sufficient for debugging, no?
```
"inlineSourceMap": true,
"inlineSources": true,
```

Regarding the issue above, the project should automatically launch the `gulp watch` task on project open, which builds the ES5. 

Finally, I pushed typings from the updated typings module. If you build from scratch, the TypeScript plugin doesn't recognize the `exlude` property correctly, and so you'll get duplicate identifier warnings like crazy. This can be resolved by manually deleting `typings/main.d.ts` and `typings/main`.

https://github.com/Foursails/visual-aurelia
